utest.h

```diff
@@ -1554,21 +1558,43 @@ int utest_main(int argc, const char *const argv[]) {
 
     ns = utest_ns();
     errno = 0;
+
+    pid_t pid = fork();
```

Comment:
We'll need to do some perf testing to see how much slower this is (I suspect it _might_ be).